lib/src/bit_counter.dart

```diff
@@ -71,8 +71,9 @@ class BitCounter {
   ///
   /// The add starts at the bit position specified by [shiftLeft].
   void addBitSet(BitSet set, {int shiftLeft = 0}) {
-    if (_length < set.length) {
-      _length = set.length;
+    // From last bit index to bit array length: +1
```

Comment:
I'm wondering if the `BitSet.length`'s definition is wrong, especially because the documentation differs on the range between `BitSet` and `BitArray`. Maybe it would be better to rename/change the field and update what it returns? wdyt?

Reply:
Correct. I think the name `length` should remain, but the value returned should be changed:
```
  int get length => _list.isEmpty ? 0 : _list.last + 1;
```
wdyt?

Reply:
Yes, I think that should be the right fix here.